spec/models/item_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Item, :type => :model do
+
+  it { is_expected.to belong_to :bag }
+
+  it { is_expected.to validate_presence_of :name }
+  it { is_expected.to validate_presence_of :bag_id }
+
```

Comment:
Please add a test to add a item to a bag and check if the length of the relation changes